src/PlayFabBuddy.Tests/PlayFabBuddy.Lib.Tests/Aggregate/PlayerAccountAggregateTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using PlayFabBuddy.Lib.Entities.Accounts;
+using Xunit;
+
+namespace PlayFabBuddy.Lib.Aggregate.Tests;
+public class PlayerAccountAggregateTests
+{
+    [Fact()]
+    public void PlayerAccountAggregateTest()
+    {
+        var mainAccount = new MasterPlayerAccountEntity
+        {
+            Id = Guid.NewGuid().ToString()
+        };
+
+        var aggregate = new PlayerAccountAggregate(mainAccount);
+
+        Assert.Equal(mainAccount, aggregate.MasterPlayerAccount);
```

Comment:
These are multiple dfifferent tests - please split up

Reply:
This test is done do make get rid of compiler warning of possible null, so i would keep it in here :)